avalon/fusion/pipeline.py

```diff
@@ -132,10 +134,18 @@ def parse_container(tool):
 
 def get_current_comp():
     """Hack to get current comp in this session"""
+    if _FORCED_COMP["COMP"] is not None:
```

Comment:
Note that when one closes the comp that was forced once it will still not equal to None with `is None`. So the invalid comp will remain. Not sure if that's intended behavior, but it's good to know that a closed comp (even though in invalid object to use) will still be returned in the case the user closed it.... it doesn't return None, it returns the invalid `PyRemoteObject`.

Reply:
> So the invalid comp will remain

Have you tested this? The only use of `force_current_comp` now is in `deadline_switch_and_submit` which is only used by Deadline.

Reply:
> The only use of force_current_comp now is in deadline_switch_and_submit which is only used by Deadline.

That's the current use case, yes. But you're exposing it in the avalon core functionality. Probably best to clarify the behavior either way... once it's in people will end up breaking this haha.